styles/Home.module.scss

```diff
@@ -18,6 +18,50 @@
   flex-direction: column;
   justify-content: center;
   align-items: center;
+  position: relative;
+  z-index: 10;
+}
+
+.hero{
```

Comment:
I'm not certain about this hero. I cannot see it with the 0.01 opactiy. If i bump that up it looks a little too in your face, it might be best to just have coming soon below the social links

Reply:
Originally meant for it to just purely decorative and not have any meaning at all since it was just a plain background. It is a good note to have the coming soon somewhere on the page <i>meant</i> to be read. Anyways, I removed it and added a "coming soon" p tag to the middle but if you want to keep it you can bring it back. As for visibility, if you want more control over how visible it should be, you can make the text color white instead but since I wanted to be pretty much invisible at first, I made it dark.

Reply:
Awesome moving it there is the best choice imo, thanks